docs/mobile-apps/virtual-usb.md

```diff
@@ -18,28 +18,234 @@ import TabItem from '@theme/TabItem';
+```
+
+```sh
+adb connect localhost:7000
+```
+
+At this point you can use `adb` with your real device. You can use Android Studio (or Google Chrome's Remote Debugging) to debug your app, execute automation based on `adb`, or any other tool that is `adb`-compliant. For example, using `adb` shell, you can start the camera of the connected device:
+
+```sh
+adb shell
+am start -a android.media.action.IMAGE_CAPTURE
+```
+
+### Connect Virtual USB to an iOS Device
+
+```sh
+d87ec06c-3803-4674-a734-05d57bfe723b		iPhone 7 Plus		IOS		14.0.1		https://app.saucelabs.com/live/mobile/dataCenters/US/devices/iPhone_7_Plus_vusb_us2/shared/d87ec06c-3803-4674-a734-05d57bfe723b
+localhost:-1	online
+```
+
+This prepares the usbmuxd socket (`/var/usbmuxd`) so that developer tools like Xcode can interact with the remote device just like they interact with a local device. You may need to have administrator permissions to replace `/var/usbmuxd` on your computer.
+
+### Closing a session created by Virtual USB client
+
+If you created your session with the Virtual USB client you have to close it so that other users can use the device.
+
+1. If you use `adb` make sure you disconnect your device from ADB using `adb disconnect <IPAddress>:<portNumber>`.
+
+```sh
+adb disconnect localhost:7000
+```
+
+2. You can close the session either by closing your Sauce Labs device view in your browser or you can run the command
 
-### Android
+```sh
+java -jar virtual-usb-client.jar deleteSession --username <USERNAME> --accessKey <ACCESS_KEY> --sessionId <SESSION_ID>
+```
+
+
+
+### Launch and Connect Virtual USB to a Real Device Live Testing Session
+
+1. Log in to Sauce Labs and get your User API key from **Sauce Apps** > **Legacy RDC** > **Account Settings** in the TestObject real device cloud platform.
+2. From a command line, launch the Virtual USB client.
+
+```sh
+java -jar client.jar server
+```
+
+If you're testing on a device in the US Data Center, you should also include the flags:
+
+```sh
+--manualUrl https://us1-manual.app.testobject.com --routerUrl wss://us1.api.testobject.com/api/vusb/forward
+```
+
+Optionally, use `-v` for verbose logging or `-vv` for super verbose logging.
+
+
+3. Return to TestObject and start a **Live Testing** session on an available Android device. The project for the session should correspond to the API key you entered in Step 2, which is the account you logged in with.
+
+4. Open a new command line terminal, and query which sessions are running.
+
+```sh
+java -jar client.jar sessions --apiKey <API KEY>
+```
+
+5. This command will return a list of open sessions, find the sessionId> and copy it. \
+
+6. Use the live testing  sessionId> to connect the Virtual USB client to the real device in your session: \
+
+```sh
+java -jar client.jar connect --sessionId <session id> --apiKey <API KEY>
+```
+
+```sh
+java -jar client.jar connect --sessionId bwerg235-5214-4f14-889c-234va9v7a --apiKey 5a0d83ab-0ccb-42ff-a822-efb23d90b55d
+```
```

Comment:
Hi @skhg @vjosafushac @stefanbirkner Thanks again for your helpful edits! Regarding Legacy vUSB: lines 170-206 are quite different from what's in the doc. Is this a new update to the TestObject workflow?

cc: @wswebcreation